ext/xsl/php_xsl.stub.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+class XSLTProcessor
+{
+    /** @return bool */
+    public function importStylesheet(object $stylesheet) {}
+
+    /** @return DOMDocument|false */
+    public function transformToDoc(DOMNode $document, ?string $return_class = null) {}
+
+    /**
+     * @param DOMDocument|SimpleXMLElement $document
+     * @return int
+     */
+    public function transformToUri($document, string $uri) {}
+
+    /**
+     * @param DOMDocument|SimpleXMLElement $document
+     * @return string|false|null
+     */
+    public function transformToXml($document) {}
+
+    /**
+     * @param string|array $name
+     * @return bool|null
+     */
+    public function setParameter(string $namespace, $name, string $value = UNKNOWN) {}
+
+    /** @return string|false */
+    public function getParameter(string $namespace, string $name) {}
+
+    /** @return bool */
+    public function removeParameter(string $namespace, string $name) {}
+
+    /** @return bool */
+    public function hasExsltSupport() {}
+
+    /**
+     * @param string|array $restrict
+     * @return void
+     */
+    public function registerPHPFunctions($restrict) {}
+
+    /** @return bool */
+    public function setProfiling(?string $filename) {}
+
+    /** @return int */
+    public function setSecurityPrefs(int $securityPrefs) {}
+
+    /** @return int */
+    public function getSecurityPrefs() {}
+}
+
+function xsl_xsltprocessor_import_stylesheet(object $stylesheet): bool {}
```

Comment:
I don't think these functions actually exist. This is just another case of an extension implement methods via functions to cause the maximum amount of confusion.

Reply:
Noo way :/ This was my first impression as well (that's why initially I converted the `FALIAS`es to `PHP_ME`s), but then I changed my mind because it seemed that these functions are exposed to users... So let's take another round :D